core/themes/seven/css/style.css

```diff
@@ -783,6 +783,36 @@ fieldset .fieldset-wrapper::before {
   border-right-color: black;
 }
 
+/**
+ * HTML5 details element.
+ */
+details {
+  padding: 6px 15px;
+  border-radius: 0 0 4px 4px;
+  border: 2px solid #EAEAEA;
+  margin: 1em 0;
+  overflow: hidden;
+  background-color: #ffffff;
+  display: table-cell;
+}
+details summary span {
+  color: #0074bd;
+  font-size: 120%;
+}
+details .details-content-wrapper,
+details .details-child-wrapper {
+  margin: 10px 0;
+}
+details.filter-wrapper summary {
```

Comment:
Same here, I don't think we need the specific style for filters, since we aren't using details for them in core.